File: Web/src/ChatRecord.js
//聊天记录类
let count = 0;

class ChatRecord {
    constructor(sender,receiver,message, date, isSend) {
      this.Sender = sender;
      this.Receiver = receiver;
      this.Message = message;
      this.Date = date;
      this.IsSend = isSend;
    }

    get Id(){
      return count++;
    }
  }

  export default ChatRecord;

File: SignalRDemo/Model/ChatRecord.cs
namespace SignalRDemo.Model
{
    public class ChatRecord
    {
        public string Sender { get; set; }
        public string Receiver { get; set; }
        public string Message { get; set; }
        public string Date { get; set; }
        public bool IsSend { get; set; }
    }
}
